src/core/rkd/core/standardlib/core.py

```diff
@@ -435,14 +435,18 @@ def configure_argparse(self, parser: ArgumentParser):
                             action='store_true')
         parser.add_argument('--no-venv', help='Do not create virtual env automatically', action='store_true')
         parser.add_argument('--pipenv', help='Generate files for Pipenv', action='store_true')
+        parser.add_argument('--latest', help='Always use latest RKD (not recommended)', action='store_true')
+        parser.add_argument('--rkd-dev', help='Development mode. Use RKD components from specified local directory')
 
     def execute(self, ctx: ExecutionContext) -> bool:
         commit_to_git = ctx.get_arg('--commit')
         without_venv = ctx.get_arg('--no-venv')
         use_pipenv = bool(ctx.get_arg('--pipenv'))
+        latest = bool(ctx.get_arg('--latest'))
```

Comment:
Mainly by the way the :create-structure task got two new switches and easier possibility to mock.